@here/harp-mapview/lib/MapViewFog.ts

```diff
@@ -5,25 +5,16 @@
  */
 
 import { Theme } from "@here/harp-datasource-protocol";
+import { Projection } from "@here/harp-geoutils";
 import { HighPrecisionLineMaterial, SolidLineMaterial } from "@here/harp-materials";
+import { assert, MathUtils } from "@here/harp-utils";
 import * as THREE from "three";
+import { MapViewUtils } from "./Utils";
 
 /**
  * Manages the fog display in [[MapView]].
  */
 export class MapViewFog {
-    private m_enabled: boolean = true;
-    private m_fog: THREE.Fog = new THREE.Fog(0x000000); // Default color asked by DefinitelyTyped.
-    private m_fogIsDefined: boolean = false;
-    private m_cachedTheme: Theme = { styles: {} };
-
-    /**
-     * Constructs a `MapViewFog` instance.
-     *
-     * @param m_scene The scene used in [[MapView]] that contains the map objects.
-     */
-    constructor(private m_scene: THREE.Scene) {}
-
```

Comment:
Why moving `enabled` getter/setter before plain properties and even before constructor?

Reply:
Auto formatter has done this ;(

Reply:
Will fix this.